app/assets/javascripts/discourse/app/instance-initializers/component-templates.js

```diff
@@ -31,11 +50,15 @@ export default {
       }
 
       let componentName = templateKey;
+      const finalOverrideModuleName = moduleNames[moduleNames.length - 1];
```

Comment:
🏌️ 
```suggestion
      const finalOverrideModuleName = moduleNames.at(-1);
```

Reply:
Good call! But will skip for now though, since I want to keep this backport as close to the `main` version as possible so that there are fewer conflicts on the next stable bump.

(and it's not worth changing in main, since we'll be dropping this initializer in a couple of weeks)